app/views/layouts/application.html.erb

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <%= render "analytics/ganalytics" %>
 
     <title>ColoradoBenefits</title>
```

Comment:
This isn't a11y-related, but we should change `<title>` to Report Changes Colorado.

Reply:
good catch!

Reply:
I made a really small change to this for now, and filed a ticket to improve across the site: https://www.pivotaltracker.com/story/show/161669616